omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -353,6 +353,16 @@
 default['private_chef']['opscode-erchef']['ssl_session_caching']['enabled'] = false
 default['private_chef']['opscode-erchef']['include_x_ops_api_info'] = false
 
+# Enable cookbook versions caching by setting this to true. If you are frequently seeing
+# very long response times from `cookbook_versions` when under load, this is worth enabling.
+default['private_chef']['opscode-erchef']['cbv_cache_enabled'] = false
```

Comment:
For now, do we want to enable by default or leave it to support to direct people to enable it when appropriate?

Reply:
It might be best to burn this in a bit first and enable it in a major later